modules/add.js

```diff
@@ -1,20 +1,14 @@
 /* eslint-disable linebreak-style */
 import Book from './book.js';
```

Comment:
- [ ] Please do not disable linters unless it is mentioned that it is not a project requirement. Linters improved your code quality and are also nice automatic debugging tools. :pray: